.github/workflows/ci.yml

```diff
@@ -97,6 +97,12 @@ jobs:
             gcr.io/iree-oss/base-arm64@sha256:942d01a396f81bff06c5ef4643ba3c9f4500a09010cd30d9ed9be691ddaf1353 \
             ./build_tools/cmake/ctest_all.sh \
             "${BUILD_DIR}"
+      - name: "Testing IREE with ArmSME"
+        # TODO: Update base-arm64 to include a build of qemu-aarch64.
+        run: |
+          ./build_tools/github_actions/docker_run.sh \
+            gcr.io/iree-oss/base-arm64@sha256:942d01a396f81bff06c5ef4643ba3c9f4500a09010cd30d9ed9be691ddaf1353 \
```

Comment:
Can you try sha256:6c0b67973771d66a23f8be7476f33b11bacdf2a51e15d394e167d54f4b12625b for base-arm64 ones?

Reply:
I missed pushing ...